app-admin/keepassxc/metadata.xml

```diff
@@ -7,9 +7,9 @@
 	</maintainer>
 	<use>
 		<flag name="autotype">Add support to autotype the passwords into other applications</flag>
-		<flag name="browser">Enables browser plugin support</flag>
+		<flag name="browser">Enable integration with keepassxc-browser</flag>
```

Comment:
old one is definitely better, the new one is more about internal impl, do not change the flag descriptions